test/check/check.go

```diff
@@ -70,6 +70,49 @@ func CheckConfigCondition(conditionType ConditionType, conditionStatus Condition
 	}
 }
 
+func CheckConfigVersions(operatorVersion, observedVersion, targetVersion string, timeout, duration time.Duration) {
+	By(fmt.Sprintf("Checking that status contains expected versions Operator: %q, Observed: %q, Target: %q", operatorVersion, observedVersion, targetVersion))
+	config := &opv1alpha1.NetworkAddonsConfig{}
+
+	getAndCheckVersions := func() error {
+		err := framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: names.OPERATOR_CONFIG}, config)
+		if err != nil {
```

Comment:
Gomega assert ?

Reply:
It is used as callback for Eventually and Consistently, therefore we gotta return something to assert later